equinix/resource_fabric_cloud_router_acc_test.go

```diff
@@ -3,103 +3,125 @@ package equinix
 				Check: resource.ComposeTestCheckFunc(
 					resource.TestCheckResourceAttr(
-						"equinix_fabric_cloud_router.test", "name", "fg_tf_acc_test"),
+						"equinix_fabric_cloud_router.test", "name", "fcr_tf_acc_test"),
 				),
 				ExpectNonEmptyPlan: false,
 			},
 			{
-				Config: testAccCloudRouterCreateConfig("fg_tf_acc_update"),
+				Config: testAccCloudRouterCreateOnlyRequiredParameterConfig("fcr_tf_acc_update"),
 				Check: resource.ComposeTestCheckFunc(
 					resource.TestCheckResourceAttr(
-						"equinix_fabric_cloud_router.test", "name", "fg_tf_acc_update"),
+						"equinix_fabric_cloud_router.test", "name", "fcr_tf_acc_update"),
 				),
 				ExpectNonEmptyPlan: false,
 			},
 		},
 	})
 }
 
-func checkCloudRouterDelete(s *terraform.State) error {
-	ctx := context.Background()
-	ctx = context.WithValue(ctx, v4.ContextAccessToken, testAccProvider.Meta().(*config.Config).FabricAuthToken)
-	for _, rs := range s.RootModule().Resources {
-		if rs.Type != "equinix_fabric_cloud_router" {
-			continue
+func testAccCloudRouterCreateOnlyRequiredParameterConfig(name string) string {
+	return fmt.Sprintf(`resource "equinix_fabric_cloud_router" "test"{
+		type = "XF_ROUTER"
+		name = "%s"
+		location{
+			metro_code  = "SV"
 		}
-		err := waitUntilCloudRouterDeprovisioned(rs.Primary.ID, testAccProvider.Meta(), ctx)
-		if err != nil {
-			return fmt.Errorf("API call failed while waiting for resource deletion")
+		package{
+			code = "LAB"
 		}
-	}
-	return nil
-}
-
-func testAccCloudRouterCreateConfig(name string) string {
-	return fmt.Sprintf(`resource "equinix_fabric_cloud_router" "test"{
-			type = "XF_GATEWAY"
-			name = "%s"
-			location{
-			  metro_code  = "SV"
-			}
-			package{
-				code = "PRO"
-			}
-			order{
-				purchase_order_number = "1-234567"
-			}
-			notifications{
-				type = "ALL"
-				emails = [
-					"test@equinix.com",
-					"test1@equinix.com"
-				]
-			}
-			project{
-				project_id = "776847000642406"
-			}
-			account {
-				account_number = 203612
-			}
-		}`, name)
+		order{
+			purchase_order_number = "1-234567"
+		}
+		notifications{
+			type = "ALL"
+			emails = [
+				"test@equinix.com",
+				"test1@equinix.com"
+			]
+		}
+		project{
+			project_id = "291639000636552"
+		}
+		account {
+			account_number = 201257
+		}
+	}`, name)
 }
 
-func TestAccCloudRouterRead(t *testing.T) {
+func TestAccCloudRouterCreateMixedParameters_PFCR(t *testing.T) {
 	resource.ParallelTest(t, resource.TestCase{
-		PreCheck:  func() { testAccPreCheck(t) },
-		Providers: testAccProviders,
+		PreCheck:     func() { testAccPreCheck(t) },
+		Providers:    testAccProviders,
+		CheckDestroy: checkCloudRouterDelete,
 		Steps: []resource.TestStep{
 			{
-				Config: testAccCloudRouterReadConfig(),
+				Config: testAccCloudRouterCreateMixedParameterConfig(),
 				Check: resource.ComposeTestCheckFunc(
 					resource.TestCheckResourceAttr(
-						"equinix_fabric_cloud_router.test", "name", "fcr_tf_acc_test"),
```

Comment:
Just like the data_source, the resource needs to have attr checks for what it's setting in response to the API call it receives after resource creation. Include those checks here as well because the resource has that schema also and we need to validate that it's functioning correctly.